backend/cli/src/main/assembly/bin/sonarlint-backend.bat

```diff
@@ -58,7 +31,10 @@ if defined JAVA_HOME_ARG (
 set CLASSPATH=%APP_HOME%\lib\*;
 
 @rem Execute SLCORE
-"%JAVA_EXE%" %DEFAULT_JVM_OPTS% -classpath "%CLASSPATH%" org.sonarsource.sonarlint.core.backend.cli.SonarLintServerCli %args%
+"%JAVA_EXE%" ^
+   %SONARLINT_JVM_OPTS% ^
```

Comment:
I tried to test passing in `SONARLINT_JVM_OPTS ` like this ` cmd.exe /c 'C:\work\sonarlint-core\backend\cli\src\main\assembly\bin\sonarlint-backend.bat -DSONARLINT_JVM_OPTS=hello'` but that kept the `DSONARLINT_JVM_OPTS` unset. What is the correct way to set it?

Reply:
I don't think it is the correct way, as it is supposed to be an environment variable, but we made a test as well with @thahnen and it was not working either. Not sure what is missing...

Reply:
I think the way to test it is more like: `export SONARLINT_JVM_OPTS=-Xmx1G && ./sonarlint-backend` or something similar in Windows

Reply:
Nice. It worked with `set SONARLINT_JVM_OPTS=-Xmx1G && C:\work\sonarlint-core\backend\cli\src\main\assembly\bin\sonarlint-backend.bat`